util/src/main/scala/scala/scalanative/util/Platform.scala

```diff
@@ -0,0 +1,5 @@
+package scala.scalanative.util
+
+object Platform {
+  def scalaVersion: String = scala.util.Properties.versionNumberString
```

Comment:
This is only used in the `tools`. Consider moving this `Platform` object to the tools, and make it package-private.

Reply:
Yes, I have to fix this.

Basically, this should change with the Scala version of the linked code that is tested, so it can't go to `tools`. This means that it should probably be refactored to be obtained via `sbt-buildinfo`. However, I can't really find an appropriate place to move it to. The only cross-compiled project `tools` depends on is `testingCompilerInterface`, but I am not sure we can enable `sbt-buildinfo` there.

Reply:
We don't have *any* project that cross compiles. There's no such thing. A project is either for the JVM or for Native. Cross compilation is an illusion provided by having two projects with the same sources. But even that we don't have in this build.

But you really don't need that. What you need is, in tools, the value of `nativelib / scalaVersion`. That's easy to obtain with sbt-buildinfo enabled on the tools project.

Reply:
But, if we enable `sbt-buildinfo` in `tools`, which is compiled with 2.12, then the `scalaVersion` in the generated sources will be 2.12. Am I missing something here?